test/python/transpiler/test_mappers.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Meta Tests for mappers
+
+Regenerate ground truth by running this on the root directory:
+> python -m  test.python.transpiler.test_mappers
+
+"""
+
+# pylint: disable=redefined-builtin,no-member,attribute-defined-outside-init
+
+import unittest
+import pickle
+
+from qiskit import ClassicalRegister, QuantumRegister, QuantumCircuit, BasicAer, compile
+from qiskit.transpiler import PassManager, transpile
+from qiskit.transpiler.passes import BasicSwap, LookaheadSwap, StochasticSwap
+from qiskit.mapper import CouplingMap
+
+from ..common import QiskitTestCase
+
+
+class CommonUtilities():
+    """ Some utilities for meta testing."""
+    regenerate_expected = False
+    seed = 42
+    additional_args = {}
+
+    def create_passmanager(self, coupling_map):
+        ''' Returns a PassManager using self.pass_class and coupling_map '''
+        return PassManager(self.pass_class(CouplingMap(coupling_map), **self.additional_args))
+
+    def create_backend(self):
```

Comment:
because they are lines that need to be repeated in every test. I prefer to keep the test as clean as possible and encapsulate the complications that are not relevant to the result. For example, if we change the backend, change it in a single place.